doc/api/stream.md

```diff
@@ -520,6 +544,9 @@ use the [`EventEmitter`][] and `readable.pause()`/`readable.resume()` APIs.
 <!--type=class-->
 
 ##### Event: 'close'
+<!-- YAML
+added: v0.3.1
+-->
```

Comment:
There should be an `added:` tag for the `stream.Readable` class, too, and since what is documented here is what is accessible via `stream.Readable`, I’d set all of these to 0.9.4, too.